www-servers/nginx/files/nginx-r2.service

```diff
@@ -7,6 +7,7 @@ Type=forking
 PIDFile=/run/nginx/nginx.pid
 RuntimeDirectory=nginx
 ExecStart=nginx
+ExecReload=kill -HUP $MAINPID
```

Comment:
I used a relative path because that's what the example from the doc does: https://www.freedesktop.org/software/systemd/man/latest/systemd.service.html#ExecReload=